src/addons/plugin/ipc.js

```diff
@@ -34,26 +42,87 @@ class IpcSocket {
     }
     this.ipcAddr = this.protocol + '://' + this.ipcFile;
 
+    this.logPrefix = 'IpcSocket' + ('   ' + this.socketId).slice(-3) + ': ' +
```

Comment:
Instead of doing all this manual padding everywhere, you can use `padStart`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart